sentry_logging/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: sentry_logging
+description: An integration which adds support for recording log from the logging package.
+version: 1.0.0
+homepage: https://docs.sentry.io/platforms/flutter/
+repository: https://github.com/getsentry/sentry-dart
+
+environment:
+  sdk: '>=2.14.4 <3.0.0'
```

Comment:
is the logging available only from 2.14.4?

Reply:
No, it's `2.12.0`.
```suggestion
  sdk: '>=2.12.0 <3.0.0'
```